pkg/build/builder/docker.go

```diff
@@ -105,10 +104,9 @@ func (d *DockerBuilder) Build() error {
 	}
 
 	if err := removeImage(d.dockerClient, buildTag); err != nil {
-		glog.Warningf("Failed to remove temporary build tag %v: %v", buildTag, err)
+		glog.Infof("warning: Failed to remove temporary build tag %v: %v", buildTag, err)
 	}
 
-	defer glog.Flush()
```

Comment:
I need to do a couple of things.  Add a flush equivalent to the signal
handler (so that on crash we at least try to dump).  Reduce the flush cycle
like we do for other commands.  And guarantee that we flush as we exit the
command (we should already have been doing this via the wrapper in
openshift).

I'm less interested in the incremental flushing - it's not worth the
abstraction.

On Sat, May 7, 2016 at 7:43 PM, Ben Parees <notifications@github.com> wrote:

> In pkg/build/builder/docker.go
> <https://github.com/openshift/origin/pull/8786#discussion_r62424313>:
>
> >  	}
> >
> > -	defer glog.Flush()
>
> why isn't this needed anymore?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8786/files/9725d5171cdd040a8f619a1a7d7b332d9228209b#r62424313>
>